RetrospectiveExtension.Frontend/components/feedbackBoardPreviewEmail.tsx

```diff
@@ -1,4 +1,4 @@
-import * as copy from 'copy-to-clipboard';
+import /* * as */ copy from 'copy-to-clipboard';
```

Comment:
Is the intent to remove import /* * as */

Reply:
removed the comment